src/transformers/dynamic_module_utils.py

```diff
@@ -152,7 +152,7 @@ def get_imports(filename: Union[str, os.PathLike]) -> List[str]:
         content = f.read()
 
     # filter out try/except block so in custom code we can have try/except imports
-    content = re.sub(r"\s*try\s*:\s*.*?\s*except\s*.*?:", "", content, flags=re.MULTILINE | re.DOTALL)
+    content = re.sub(r"\s*try\s*:.*?\s*except.*?:", "", content, flags=re.MULTILINE | re.DOTALL)
```

Comment:
```suggestion
    content = re.sub(r"\s*try\s*:.*?except.*?:", "", content, flags=re.MULTILINE | re.DOTALL)
```

Reply:
Looks good to me - can we take this even further and remove the whitespace matching here as well? With `re.DOTALL` they should be equivalent. Also, `re.MULTILINE` only affects [the handling of start and end characters](https://docs.python.org/3/library/re.html#re.MULTILINE), which aren't used here. Maybe we can drop that as well?

Reply:
Yes, definitely. I removed  `re.MULTILINE` and extra `\s*`. Tests are fine.